lazer/sdk/js/examples/index.ts

```diff
@@ -1,13 +1,21 @@
 import { PythLazerClient } from "../src/index.js";
 
-/* eslint-disable no-console */
-const client = new PythLazerClient("ws://127.0.0.1:1234/v1/stream", "ctoken1");
+// Ignore debug messages
+console.debug = () => {};
```

Comment:
I think this can cause issues for the end user. let's avoid overriding global stuff.

Reply:
oh actually this is the example so it's fine I guess